tests/testthat/test-multitables.R

```diff
@@ -228,6 +230,12 @@ with_test_authentication({
         expect_identical(names(refresh(multitables(ds))), "Yet another name")
     })
 
+    test_that("Can copy the multitable to a new multitable", {
+        m <- importMultitable(name='copied_multitable', data=ds, multitable=mult)
+        expect_identical(name(m), "copied_multitable")
+        is.public(multitables(ds))[2] <- TRUE
```

Comment:
i wanted to maintain that a public multitable when copied remained public, so it was an intentional copy.  Also, it verifies that the multitable object actually exists.